packages/server/src/worker.ts

```diff
@@ -78,6 +85,13 @@ async function start() {
           );
         case JobKind.Test:
           return await processJobTest(job as Job<TJobTestData, TJobTestResult>);
+        case JobKind.SyncAlgolia:
+          return await processJobSyncAlgolia(
+            job as Job<TJobSyncAlgoliaData, TJobSyncAlgoliaResult>,
+            ALGOLIA_APP_ID,
```

Comment:
I passed these variables in to keep env variable gathering all in the same place - but I really debated this so let me know what you think is best!

Reply:
I love it.

The only question I have is if the Algolia envs aren't there would the worker throw on start or only when a sync algolia job is pulled out of the queue. (I'm pretty sure it's on a job) The tradeoff is if it fails at start we'll def know there's something wrong. But on the other hand, we don't really want to put envs into travis, and if it fails without the envs at start then we'd have to even if we weren't testing this specifically. We don't have worker tests right now but this bit me with the server e2e tests.

Since we're not really testing the worker this is fine, but see this PR of a possible solution if interested https://github.com/CodeForBoulder/upswyng/commit/4ad4e5ad1ec971b482809e8ea1c270d0438ee4b8.